src/lib/components/ContentfulRichText/__tests__/documents/markdown-document.md

```diff
@@ -0,0 +1,120 @@
+<!-- prettier-ignore-start -->
+
+# h1 Heading
+
+## h2 Heading
+
+### h3 Heading
+
+#### h4 Heading
+
+##### h5 Heading
+
+###### h6 Heading
+
+## Paragraphs
+
+This is a paragraph
+with a new line.
+
+This is a new paragraph.
+
+This is a paragraph<br/>using br.
+
+## Horizontal Rules
+
+---
+
+---
+
+---
+
+## Emphasis
+
+**This is bold text**
+
+**This is bold text**
+
+_This is italic text_
+
+_This is italic text_
+
+~~Strikethrough is not supported~~
+
+## Blockquotes
+
+> Blockquotes
+
+## Lists
+
+Unordered
+
+- Create a list by starting a line with `+`, `-`, or `*`
+- Sub-lists are made by indenting 2 spaces:
+  - Marker character change forces new list start:
+    - Ac tristique libero volutpat at
+    * Facilisis in pretium nisl aliquet
+    - Nulla volutpat aliquam velit
+- Very easy!
+
+Ordered
+
+1. Lorem ipsum dolor sit amet
+2. Consectetur adipiscing elit
+3. Integer molestie lorem at massa
+
+4. You can use sequential numbers...
+5. ...or keep all the numbers as `1.`
+
+Start numbering with offset:
+
+57. foo
```

Comment:
Dumb question, but we're not expecting this to start the count at 57, right?

Reply:
It's not supported by `@contentful/rich-text-from-markdown`, so I don't see a need for us to support it unless we find out we have to.